### PR TITLE
[SCRUM-31] relevant survey

### PR DESCRIPTION
**Pull request by Eddyhzz**

### CODE REVIEW

File: src/moral-survey-4/img/WechatIMG1017.jpg
Comment (Xinlong-Wu) [inline src/moral-survey-4/img/WechatIMG1017.jpg:1]: change to a clear name

File: src/templates/understandTopic.html [hunk 1]
```diff
@@ -49,6 +49,9 @@ <h2 class="bolded-blue" data-i18n="moral-topic-understand-{{topic}}">###</h1>
                 <p class="bolded-blue fs-5" data-i18n="moral-practice-skip"></p>
                 <button type="button" class="btn btn-secondary me-auto" onclick="skipTraing()" data-i18n="moral-skip"></button>
             </div>
+
+            {{#if isRele}}
+            {{else}}
```
Comment (Xinlong-Wu) [inline src/templates/understandTopic.html:54]: please remove it if `if` statment is empty

File: src/moral-survey-4/img/WechatIMG1017.jpg
Comment (Eddyhzz) [reply src/moral-survey-4/img/WechatIMG1017.jpg:1]: ok

File: src/templates/understandTopic.html (hunk 1 above)
Comment (Eddyhzz) [reply src/templates/understandTopic.html:54]: ok

**Pull request merged**